lib/bike_brigade_web/live/rider_live/index.ex

```diff
@@ -52,7 +52,7 @@ defmodule BikeBrigadeWeb.RiderLive.Index do
 
           %__MODULE__{active: actives}
 
-        ["capacty", capacity] ->
+        ["capacity", capacity] ->
```

Comment:
?

Reply:
Good catch, this is indeed breaking the autocomplete suggestions
<img width="745" alt="Screen Shot 2022-04-05 at 4 16 54 PM" src="https://user-images.githubusercontent.com/34720/161841829-ce98cd2d-e1bd-4392-9c86-a9630c382185.png">
<img width="742" alt="Screen Shot 2022-04-05 at 4 17 14 PM" src="https://user-images.githubusercontent.com/34720/161841822-3c876559-aad4-4f02-bc1d-9a6717d01ec8.png">